pkg/apis/fake/v1/db_types.go

```diff
@@ -43,6 +43,9 @@ type DBSpec struct {
 type OperationFlagType string
 
 func (o OperationFlagType) Enabled() bool {
+	if o == "" {
```

Comment:
Is this needed for this PR? Otherwise, please put it in a dedicated PR

Reply:
will add in separate PR